lib/tasks/update_resource_quantity.rake

```diff
@@ -0,0 +1,10 @@
+namespace :data do
+  desc 'Update resources with nil quantity'
+  task update_resource_quantity: :environment do
+    Resource.where('used is null or quantity is null').each do |resource|
+      resource.used = 0 unless resource.used.present?
+      resource.quantity = resource.used unless resource.quantity.present?
+      puts 'Failed for resource with id ' + resource.id.to_s + ' and name ' + resource.name if !resource.save
```

Comment:
How about the following? It is a bit more readable as this line is getting too long.

```
unless resource.save
  puts "Failed to update resource #{resource.name} (ID #{resource.id})"
end
```